src/main/java/uk/gov/hmcts/divorce/citizen/event/CitizenSubmitApplication.java

```diff
@@ -81,9 +75,26 @@ public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuild
                 .state(Draft)
                 .build();
         }
+
+        State state;
+        if (caseDataCopy.getHelpWithFeesAppliedForFees() != null
```

Comment:
if caseDataCopy.getHelpWithFeesAppliedForFees() is null what should be the default state ?

Reply:
from the FE application stand, it should never be null, you have to answer that question. but here, included it, i can remove it as that field will be hit in the FE

Reply:
from backend perspective it will be always set either to yes or no so should be never null